OrbitQt/servicedeploymanager.cpp

```diff
@@ -85,7 +85,7 @@ outcome::result<bool> ServiceDeployManager::CheckIfInstalled() {
     version = version.substr(1);
   }
   const auto command =
-      absl::StrFormat("/usr/bin/dpkg-query -W orbitprofiler 2>/dev/null | grep %s", version);
+      absl::StrFormat("/usr/bin/dpkg-query -W orbitprofiler 2>/dev/null | grep -E '%s$'", version);
```

Comment:
I am unsure if this `-E '%s$'` does exactly what you want. `version` is a simple (not escaped) std::string, right? So for an example of `version` would be "1.52", which results in `grep -E '1.52$'`. This would then also match i.e. "1352". 
`echo "1352" | grep -E '1.52$'`

Reply:
Yes. But the problem has been there before and is unrelated to that change. But yeah, maybe we should fix it.

Reply:
I've just pushed a different solution. PTAL. Thanks.